auditlog_security/models/auditlog_log_line.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022-2024 Therp B.V.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.osv.expression import OR
+
+
+class AuditlogLogLine(models.Model):
+    _inherit = "auditlog.log.line"
+    _order = "create_date desc"
+
+    user_id = fields.Many2one(related="log_id.user_id")
+    method = fields.Char(related="log_id.method")
+    model_id = fields.Many2one(related="log_id.model_id", store=True, reaonly=True)
```

Comment:
```suggestion
    model_id = fields.Many2one(related="log_id.model_id")
```

I still strongly think those fields shouldn't be stored, and related fields are readonly by default

Reply:
OK. Since we use model_id for filtering in some tree views, shall I just create a computed and stored field stored_model_id and use that in the views? Or make this one searchable?

Reply:
basically we do not even need that field anymore, removing